lib/safira_web/controllers/redeemable_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule SafiraWeb.RedeemableController do
+  use SafiraWeb, :controller
+
+  alias Safira.Contest
+  alias Safira.Accounts.User
+  alias Safira.Accounts
+
+  action_fallback SafiraWeb.FallbackController
+
+  def index(conn, _params) do
+      redeemables = Contest.list_redeemables()
+      render(conn, "index.json",redeemables: redeemables)
+  end
```

Comment:
Since there isn't a view yet implemented, trying to call `render` will return `internal server error`. Don't think it's a very good idea to commit code to the main branch that could yield errors, do you mind implementing a quick view with the render function? 

What's your opinion on this @joaop21

Reply:
I forget to push this, it was on 'limbo' 🤦‍♂️